### ISSUE TITLE
Feature request: tabbed searches + GUI closure

### ISSUE THREAD

**Issue by wellloaded** [Jackett/Jackett#7739]
1) Tabbed searches
I usually run multiple searches at the same time and to do so I just open extra tabs in my browsers.
However this might become unpractical when you have many of these tabs open so I was wondering:

Would you consider redesign the search window so that you can run multiple queries at the same time (one tab per search).

2) Allow only top right X to close search window
Also I always get caught by mistakenly clicking just outside the search overlay window (see pic below) which by design as I understand closes the search itself. In my case 99% of the times this is unwanted, would you consider removing this function and allow only the traditional X in the top right corner to close that window?

![image](https://user-images.githubusercontent.com/46318040/77144691-d8949700-6a86-11ea-8678-454709ca5787.png)

Regardless wonderful project you're really doing a great job here :-)

Thanks!!!

**Comment by RoloSoze** [comment]
I support both suggestions, especially the second. Accidental closures happen to me all the time.

**Comment by cadatoiva** [comment]
For number 2:
From all my debugging, closing the popup does not stop the underlying search (currently) so if you wait some time for the search to complete (if it hadn't before close) and click the "View Cached Releases" button, you can get the search back from in there.

Just pointing out a temporary work around until it gets implemented.

**Comment by RoloSoze** [comment]
> From all my debugging, closing the popup does not stop the underlying search (currently) 

Yeah, I noticed that. I think it would be a good idea to have a 'Pause Search' button in case the user changes their mind during a search. When pressed, Jackett should stop searching and display the results retrieved up until that point.

**Comment by cadatoiva** [comment]
Something like that won't be able to be implemented without massive changes to the existing code. We're only a couple of people, so it's going to be a while. However, current changes I'd like to implement will make adding this kind of thing much easier.
